frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipeline-task-runs-display.feature

```diff
@@ -58,9 +58,23 @@ Feature: Display task runs page
               And user will see "<workspace_type>" label with "<workspace_name>" Workspace "shared-task-storage" mentioned in the "<resource>" Resources section of Task Run Details page
 
         Examples:
-                  | workspace_type | workspace_name | resource            |
-                  | PVC            | PVC            | Workspace           |
-                  | CM             | Config Map     | Workspace           |
-                  | S              | Secret         | Workspace           |
-                  |                | Empty Directory| Workspace           |
-                  | PVC            | PVC            | VolumeClaimTemplate |
+                  | workspace_type | workspace_name  | resource            |
+                  | PVC            | PVC             | Workspace           |
+                  | CM             | Config Map      | Workspace           |
+                  | S              | Secret          | Workspace           |
+                  |                | Empty Directory | Workspace           |
+                  | PVC            | PVC             | VolumeClaimTemplate |
+
+
+        @regression @to-do
+        Scenario: Task Run results on Task Run details page for passed task run
+            Given user is on Task Run details page of passed task run
+             When user scrolls to the Task Run results section
+             Then user can see Name and Value column under Task Run results
+
+
+        @regression @to-do
+        Scenario: Task Run results on Task Run details page for failed task run
+            Given user is on Task Run details page of failed task run
```

Comment:
Is it possible to provide test data for failed task run, If not mark this scenario as @manual

Reply:
Updated